### PR TITLE
Tuning clean up

### PR DESCRIPTION
**Pull request by EthanMarx**
- Integrating with the latest `lightray` update which now exposes most of the `tune.Tuner` API. This will allow users to configure things like the `TrialScheduler`, `SearchAlg` etc.
- Adds `TuneLocal` task which works the same way as `TrainLocal` vs `TrainRemote`: If `AFRAME_TRAIN_RUN_DIR` is an s3:// path will tune remotely.
- Removes `TraceModel` callback, which was unnecessary to do at the end of every epoch for every trial, and was contributing to some bottlenecks. Don't yet have a cleaner solution for automatically tracing the best model, so will have to do it by hand for now.

### CODE REVIEW

File: projects/train/configs/bns.yaml [hunk 1]
```diff
@@ -117,4 +116,5 @@ trainer:
   max_epochs: 200
   check_val_every_n_epoch: 1
   log_every_n_steps: 20
+  enable_progres_bar: true
```
Comment (wbenoit26) [inline projects/train/configs/bns.yaml:119]: I'm guessing this should be "progress"?

**Review by wbenoit26 — CHANGES_REQUESTED** [review]
Looks good, I think there's just a typo in `bbh.yaml` and `bns.yaml`

File: projects/train/configs/bns.yaml (hunk 1 above)
Comment (EthanMarx) [reply projects/train/configs/bns.yaml:119]: yup

**Review by wbenoit26 — DISMISSED** [review]

**Comment by EthanMarx** [comment]
> Looks good, I think there's just a typo in `bbh.yaml` and `bns.yaml`

What was the typo in `bbh.yaml`?

**Comment by wbenoit26** [comment]
I thought it was also "progres" instead of "progress", but it looks like I misread it, or just looked at `bns.yaml` twice

**Comment by EthanMarx** [comment]
@wbenoit26 can I get an approval when you get a chance?

**Pull request merged**